### ISSUE TITLE
hard to reconnect to peers

### ISSUE THREAD

**Issue by colinhd8** [rakshasa/rtorrent#338]
It seems that rtorrent hard to reconnect to the peers after restart the torrent.
1.load a torrent and start it.
2.After some peers connected(this time it can connect to the peers quickly), stop the torrent.(don't exit rtorrent)
3.After a while, start the torrent.Then it will have no speed and can't connect to any peer for a long time. 
if don't stop the torrent and restart rtorrent, it can connect to the peers quickly.

OS: Fedora 22
rtorrent: 0.9.6

**Comment by xbtowbw** [comment]
Yeah I noticed this as well. It takes rtorrent very long to resume a once stopped download. Is that bug? What can be done against?

**Comment by mrvn** [comment]
The problem I see is that rtorrent does not remember the available peers when you stop a torrent. On restart it asks the tracker for new peers again but the tracker won't hand out the same peers as the last time (if the torrent was not stopped long). Often I get no peers at all. Only after a long while the tracker will forget what peers it told me already and give me peers again.

**Comment by colinhd8** [comment]
I think it's the reason.
